configs/opa_bundles/default/authz.rego

```diff
@@ -62,6 +77,25 @@ allow if {
 	input.action == "create"
 }
 
+# new members must be invited.
+allow if {
+	authn
+	input.kind == "org"
+	input.action == "add-member"
+	is_org_member_of_resource
+	input.data.status == "INVITED"
```

Comment:
Does this mean we can't auto/forcefully add users to an org? Does this conflict with the `--active` flags in the `add-member` CLI command?